code/modules/antagonists/voidwalker/voidwalker_organs.dm

```diff
@@ -107,3 +123,17 @@
 /obj/effect/spawner/glass_shards/mini
 	min_spawn = 1
 	max_spawn = 2
+
+/obj/effect/spawner/glass_debris
+	/// Weighted list for the debris we spawn
```

Comment:
Why does this not subtype `/obj/effect/spawner/random`?

That does everything this subtype is aiming to do (spawn a random item from a weighted loot pool) (it's what maint spawners use)

Reply:
uhhhh I dont know. Good suggestions!